### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by rabayarabu**
Use basic JavaScript syntax.
Use JavaScript to manipulate DOM elements.
Use JavaScript events.
Understand how to parse a Figma design to create a UI.
Use objects to store and access data.
When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

File: index.html
Comment (AbelG101) [inline index.html:1]: https://github.com/rabayarabu/Portfolio-setup-and-mobile-first/blob/3335e5333c14bdec649436bfc1f765ebfce1567d/index.html#L65-L89
- [ ] The above section is categorized as one of the projects as well so please make sure to remove it & only load it dynamically like the rest of the projects.
- [ ] Also please make sure to implement the popup for that project just like the rest of the projects. ✔

File: style.css [hunk 1]
```diff
@@ -698,94 +636,95 @@ i.close.show {
...
+  top: 49%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: #fff;
+  z-index: 11;
+  max-height: 98vh;
+  border-radius: 10px;
+  margin: 20px auto;
+  margin-bottom: 20px;
+}
```
Comment (AbelG101) [inline style.css]: - [ ] The popup is looking wonderful but there seems to be a bit of a discrepancy with the Figma design, let me clarify below: 👇
- Figma design:
  image
- Your design:
  Inkedfirefox_E4EeSRhDPE
   - [ ] As I've highlighted in blue above, please fix the spacing issues by comparing it with the Figma design
   - [ ]  Also make sure to follow the proper font family as specified in the Figma design. (The font for the entire text in the popup section needs to be modified)

File: style.css [hunk 2]
```diff
@@ -1126,77 +956,44 @@ i.close.show {
...
+    justify-content: space-between;
+    gap: 10px;
+    width: 140px;
+    color: #fff;
+    background: #ff6b00;
+    border: none;
+    padding: 12px;
+    font-size: 15px;
+    cursor: pointer;
   }
```
Comment (AbelG101) [inline style.css]: - [ ] Please make sure to follow the Figma design for the different states of the button here as I've noticed you forgot to implement the `:hover` and `:active` states.
- [ ] Here's the design as a reminder 😉
  firefox_IJ0QIflmz9

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @rabayarabu 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- You just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @rabayarabu, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**